File: MobileWeather/WeatherLocation.m
//
//  WeatherLocation.m
//  MobileWeather
//
//  Copyright (c) 2013-2015 Ford Motor Company. All rights reserved.
//

#import "WeatherLocation.h"

@implementation WeatherLocation

- (instancetype)initWithCity:(NSString *)city state:(NSString *)state zipCode:(NSString *)zip country:(NSString *)country gpsLocation:(CLLocation *)location {
    self = [super init];
    if (!self) { return nil; }

    _city = city;
    _state = state;
    _zipCode = zip;
    _country = country;
    _gpsLocation = location;

    return self;
}

@end

File: MobileWeather/WeatherLocation.swift
//
//  WeatherLocation.swift
//  WeatherLocationService
//
//  Created by Joel Fischer on 6/7/21.
//  Copyright © 2021 Ford. All rights reserved.
//

import CoreLocation
import Foundation

public struct WeatherLocation {
    let country: String?
    let state: String?
    let city: String?
    let zipCode: String?
    let gpsLocation: CLLocation
}
